search-sentiments-map/public/scripts/trends-script.js

```diff
@@ -19,26 +19,29 @@ const TOGGLE_SHOW_LESS = 'Show Less';
 const CLASSNAME_SHOWN = 'shown';
 const CLASSNAME_HIDDEN = 'hidden';
 
-/** Displays the top trends of the US on the DOM. */
+/** Displays the top trends on the DOM. */
 function setTopTrends() {
   const trendsList = document.getElementById('trends-list');
 
-  // Get the 20 trending search topics from the backend.
+  // Get the top 10 globally trending search topics from the backend.
   fetch('/trends').then(globalTrends => globalTrends.json()).then(trends => {
     for (let i = 0; i < trends.length; i++) {
       const trendElement = document.createElement('li');
-      trendElement.innerText = `${trends[i].trendTopic} (${trends[i].country})`;
-      // Show 10 trending topics by default and hide the rest.
-      trendElement.className = i < 10 ? CLASSNAME_SHOWN : CLASSNAME_HIDDEN;
+      trendElement.innerText = `${trends[i].trendTopic}`;
+      // Display the number of countries where the trend is trending when hovered.
+      trendElement.title = `Trending in ${trends[i].count} countries`;
+
+      // Show 7 trending topics by default and hide the rest.
+      trendElement.className = i < 7 ? CLASSNAME_SHOWN : CLASSNAME_HIDDEN;
```

Comment:
If we only have 10 total, why don't we just list all 10 and get rid of the show more/less button.

Reply:
We could add more if the search limit allows, or actually add some unclickable trends (I just want to keep the more/less button because it was a lot of work lol